docs/source/release/version0.13.0.rst

```diff
@@ -88,6 +88,12 @@ hypothesis test for stochastic equality in
 confidence intervals and equivalence testing (TOST) for the stochastically
 larger statistic, also known as Common Language effect size.
 
+Intervening Variable Analysis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+:class:`~statsmodels.stats.intervening_variable.InterveningVariables` creates 
+confidence intervals for the indirect effect using Sobel's classical method 
+as well as bootstrapping techniques.
+
```

Comment:
this file is for 0.13 which was already released

This PR is a candidate for 0.15

Reply:
I don't see any release notes after 0.13. Should I add a document?

Reply:
The release notes are mostly autogenerated. So we don't add them until a release.

You could park this section at the bottom of the top comment of the PR. Then we can copy it over when we have the release note file.

(We should find some better way to add to release notes, but don't have one yet)